zerver/tests/test_user_groups.py

```diff
@@ -652,7 +669,7 @@ def test_query_counts(self) -> None:
             for i in range(50)
         ]
 
-        with self.assert_database_query_count(4):
+        with self.assert_database_query_count(11):
             user_group = create_user_group_in_database(
                 name="support",
                 members=[hamlet, cordelia, *original_users],
```

Comment:
Should we also add a test to assert query count for case when single user group is not created?

Reply:
I have added test for query counts for the case when  single user group is not created.